pages/2_subscriptions_report.py

```diff
@@ -16,34 +17,113 @@
 ## Define data and filters. The resulting data variable includes the data with all filters applied.
 data = page_creation()
 
-st.divider()
-
-st.dataframe(data.head(10)) ## This is here just as an example. This can be deleted during development and before release.
-
-st.divider()
-
```

Comment:
Let's keep the one divider here to ensure the filters have a divider between the charts.